packages/paypal-commerce-integration/src/paypal-commerce-fastlane/paypal-commerce-fastlane-customer-strategy.spec.ts

```diff
@@ -462,5 +462,25 @@ describe('PayPalCommerceFastlaneCustomerStrategy', () => {
 
             expect(executionOptions.continueWithCheckoutCallback).toHaveBeenCalled();
         });
+
+        it('doesnt skip shipping step if isEnableFastlaneShippingOptionAutoSelect is false', async () => {
```

Comment:
```suggestion
        it('doesnt select shipping option if isFastlaneShippingOptionAutoSelectEnabled is false', async () => {
```

Reply:
May I ask you to add (or update existing one) a test when isFastlaneShippingOptionAutoSelectEnabled is true?